.travis.yml

```diff
@@ -32,6 +32,10 @@ matrix:
       env:
         - SYMFONY_VERSION='2.7.*@dev'
         - FRAMEWORK_EXTRA_VERSION='~3.0'
+    - php: 5.6
+      env:
+        - PHPUNIT_FLAGS="--coverage-clover=coverage.clover"
+        - COVERAGE=true
 
 # only needed for the hackaround for https://github.com/symfony/symfony/issues/12868
```

Comment:
afaik there is now an official method for this: http://symfony.com/blog/new-in-symfony-2-7-phpunit-bridge 

do you want to try that as well? so we could clean up some of the tricks we did to avoid problems with the deprecation notices.

@WouterJ i thought we are using this in the cmf now, but can't find the right place to point out. can you help us find the example?